R/exportToJson.R

```diff
@@ -619,9 +621,18 @@ generateDomainMetaReport <- function(conn, dbms, cdmDatabaseSchema, resultsDatab
                                             vocab_database_schema = vocabDatabaseSchema
   )  
   
-  output$MESSAGES <- querySql(conn,queryDomainMeta)
-  jsonOutput = toJSON(output)
-  write(jsonOutput, file=paste(outputPath, "/domainmeta.json", sep=""))  
+  try(
+    output$MESSAGES <- querySql(conn,queryDomainMeta)
+  )
+  if (output$MESSAGES = NULL)

```

Comment:
This line seems to result in an error on the build server.  Is it supposed to use == or is there an isNull operator?